src/cpp/CMakeLists.txt

```diff
@@ -117,11 +117,11 @@ set(${PROJECT_NAME}_source_files
     fastdds/domain/DomainParticipant.cpp
     fastdds/domain/qos/DomainParticipantQos.cpp
     fastdds/domain/qos/DomainParticipantFactoryQos.cpp
-    fastdds/builtin/typelookup/common/TypeLookupTypes.cpp
-    fastdds/builtin/common/RPCHeadersImpl.cpp
-    fastdds/builtin/typelookup/TypeLookupManager.cpp
-    fastdds/builtin/typelookup/TypeLookupRequestListener.cpp
-    fastdds/builtin/typelookup/TypeLookupReplyListener.cpp
+    fastdds/builtin/typelookupservice/common/TypeLookupTypes.cpp
+    fastdds/builtin/typelookupservice/common/RPCHeadersImpl.cpp
```

Comment:
The folder `common` is going to disappear, leaving the types in the same folder. This can be done in the follow-up PR that generates the types from the IDL.

Reply:
I think it makes sense to move all the related files together now, and then remove them in the idl's PR